### ISSUE TITLE
Make Piwik usable with keyboard: implement proper tabindex order

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#9562]
The goal of this issue is to make Piwik much more usable with a keyboard. In particular, it should be possible to "tab" through various elements on screens using the keyboard.

What we expect is that, when using the TAB key, it should tab through the following elements on the screen:

## Proposed tab order when using TAB key

 * `Skip to main content`
 * then... the Search bar
 * then... the Piwik logo
 * then... the top menu
 * then... the Website selector
 * then... the Calendar
 * then... the Segment editor 
 * then... the Dashboard/Widgets selector
 * then... the main navigation left menu
 * then... the main content / report / widgets

## Info

http://webaim.org/techniques/keyboard/tabindex
